components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java

```diff
@@ -526,15 +526,33 @@ public static GovernanceArtifactConfiguration getGovernanceArtifactConfiguration
                     // if no permission definitions were present, define the default ones.
                     List<ManagementPermission> managementPermissions =
                             new LinkedList<ManagementPermission>();
-                    String idPrefix = "/permission/admin/manage/resources/govern/" +
-                            configuration.getKey();
+                    String idPrefixManage = "/permission/admin/manage";
+                    String idPrefixResources = idPrefixManage + "/resources";
+                    String idPrefixGovern = idPrefixResources + "/govern";
+                    String idPrefixFromConfig = idPrefixGovern + "/" + configuration.getKey();
+                    String idPrefixConfigure = "/permission/admin/configure";
+                    String idPrefixGovernance = idPrefixConfigure + "/governance";
+
+                    managementPermissions.add(
```

Comment:
Can you please review the line length? It seems this code is formatted with 100 column space.